src/HDInsight/HDInsight/help/Enable-AzHDInsightMonitoring.md

```diff
@@ -1,31 +1,31 @@
-﻿---
+---
 external help file: Microsoft.Azure.PowerShell.Cmdlets.HDInsight.dll-Help.xml
 Module Name: Az.HDInsight
-online version: https://docs.microsoft.com/en-us/powershell/module/az.hdinsight/enable-azhdinsightoperationsmanagementsuite
+online version: https://docs.microsoft.com/en-us/powershell/module/az.hdinsight/enable-azhdinsightmonitoring
 schema: 2.0.0
 ---
 
-# Enable-AzHDInsightOperationsManagementSuite
+# Enable-AzHDInsightMonitoring
 
 ## SYNOPSIS
-Enables Operations Management Suite (OMS) in a HDInsight cluster and relevant logs will be sent to the OMS workspace specified during enable.
+Enables monitoring in a HDInsight cluster and relevant logs will be sent to the monitoring workspace specified during enable.
 
 ## SYNTAX
 
 ```
-Enable-AzHDInsightOperationsManagementSuite [-Name] <String> [-WorkspaceId] <String> [-PrimaryKey] <String>
+Enable-AzHDInsightMonitoring [-Name] <String> [-WorkspaceId] <String> [-PrimaryKey] <String>
  [-ResourceGroupName <String>] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
  [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The **Enable-AzHDInsightOperationsManagementSuite** cmdlet enables Operations Management Suite (OMS) in a Azure HDInsight cluster.
+The **Enable-AzHDInsightMonitoring** cmdlet enables monitoring in a Azure HDInsight cluster.
 
 ## EXAMPLES
 
 ### Example 1
 ```
-PS C:\> Enable-AzHDInsightOMS -Name testcluster -WorkspaceId 1d364e89-bb71-4503-aa3d-a23535aea7bd -PrimaryKey <key for workspace 1d364e89-bb71-4503-aa3d-a23535aea7bd>
+PS C:\> Enable-AzHDInsightMonitoring -Name testcluster -WorkspaceId 1d364e89-bb71-4503-aa3d-a23535aea7bd -PrimaryKey <key for workspace 1d364e89-bb71-4503-aa3d-a23535aea7bd>
 
 ErrorInfo  :
```

Comment:
@aim-for-better same comment about updating the output type in the examples in this file as well as the output type at the bottom of the file

Reply:
> @aim-for-better same comment about updating the output type in the examples in this file as well as the output type at the bottom of the file

updated